server/src/main/java/org/spine3/server/aggregate/Aggregates.java

```diff
@@ -68,95 +59,39 @@ private Aggregates() {
      *                  // The expected constructor:
      *                  CustomAggregatePart(AnAggregateId id, CustomAggregateRoot root){...}
      *              }
-     *
      *          }
      *          </pre>
-     *      </li>
-     * </ul>
      *
      * <p>Throws {@code IllegalStateException} in other cases.
      *
      * @param entityClass the {@code AggregatePart} class
-     * @param idClass     the class of entity identifiers
      * @param <E>         the entity type
      * @param <I>         the ID type
      * @return the {@code AggregatePart} constructor
      * @throws IllegalStateException if the entity class does not have the required constructor
      */
-    static <E extends AggregatePart, I, R extends AggregateRoot<I>> Constructor<E>
-    getAggregatePartConstructor(Class<E> entityClass, Class<R> rootClass, Class<I> idClass) {
+    static <E extends AggregatePart<I, ?, ?, ?>, I> Constructor<E>
```

Comment:
It would be better to have `A` instead of `E` for consistency with the class declaration.